app/controllers/zendesk/drop_offs_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Zendesk
+  class DropOffsController < ApplicationController
+    include ZendeskAuthenticationControllerHelper
+    include FileResponseControllerHelper
+
+    before_action :require_zendesk_user, :set_drop_off, :require_ticket_access
+
+    def show
+      render_active_storage_attachment @drop_off.document_bundle
+    end
+
+    private
+
+    def set_drop_off
+      @drop_off = IntakeSiteDropOff.find(params[:id])
+    end
+
+    def zendesk_ticket_id
+      @drop_off.zendesk_ticket_id
```

Comment:
we likely need the same Zendesk instance protection that we use on the documentscontroller. Moreso on this one if anything because there are more older tickets.

Reply:
but if we haven't stored the instance then don't worry about it.

Reply:
unfortunately we have not stored the instance (probably because UWTSA had their own system for valet)